tests/tool/Integration.Tests/IntegrationScenarios/WCFSample/Upgraded/wcf.config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <system.serviceModel>
     <services>
-      <service name="WcfServiceLibrary1.Service1">
+      <service name="WcfServiceLibrary1.Service1" behaviorConfiguration="">
         <!--The host element is not supported in configuration in CoreWCF. The port that endpoints listen on is instead configured in the source code.-->
```

Comment:
Does everything still work when behaviorConfiguration isn't specified as well as the behavior name attribute? The usual case will be them not being specified in the config file.

Reply:
When the behavior name is not specified, I will set them to be empty string so they can still be passed into different methods. It's done in the SetServiceAndBehaviorNames() method.